Firestore/tests/Unit/Connection/GrpcTest.php

```diff
@@ -26,7 +26,7 @@
 use Google\Cloud\Firestore\V1\DocumentMask;
 use Google\Cloud\Firestore\V1\Precondition;
 use Google\Cloud\Firestore\V1\StructuredQuery;
-use Google\Cloud\Firestore\V1\StructuredQuery_CollectionSelector;
+use Google\Cloud\Firestore\V1\StructuredQuery\CollectionSelector;
 use Google\Cloud\Firestore\V1\TransactionOptions;
 use Google\Cloud\Firestore\V1\TransactionOptions_ReadWrite;
```

Comment:
may change this too

Reply:
Yep, changed in commit [41b2311](https://github.com/googleapis/google-cloud-php/pull/5865/commits/41b2311e14606643954616150399eaf1a13d0ba1) and it's working.